manifests/init.pp

```diff
@@ -182,7 +182,7 @@
   Array[String] $bindcmdaddress                                    = ['127.0.0.1', '::1'],
   Array[String] $cmdacl                                            = $chrony::params::cmdacl,
   Optional[Stdlib::Port] $cmdport                                  = undef,
-  $commandkey                                                      = 0,
+  Integer $commandkey                                              = 0,
```

Comment:
Is there a lower/upper bound? Like `Integer[0]`?

Reply:
Well, chrony.conf(5) states ```the range 1 through 2^32-1```, which should mean that we
could use Integer[1,4294967295] but since we are using 0 as default I didn't dare to
change that. See #91

Reply:
https://chrony.tuxfamily.org/faq.html#_what_happened_to_the_code_commandkey_code_and_code_generatecommandkey_code_directives

Reply:
Thanks, I think we should continue to allow 0 then.  Perhaps with ```Integer[0,4294967295]``` ? Or just Integer[0] ?

Reply:
Suggest just `Integer[0]` and leave the upper bound constraints to chrony.